seungtae/app.js

```diff
@@ -0,0 +1,52 @@
+const dotenv = require("dotenv");
+dotenv.config();
```

Comment:
```suggestion
require("dotenv").config();
```
다음과 같이 한줄로 작성하면 코드도 짧고 간결하면서, 똑같이 동작합니다. 기존 코드와 비교해보세요.